index.html

```diff
@@ -0,0 +1,273 @@
+           </header>
+           <main>
+                <section class="first-screen row" id="about">
+                    <div class="col-2 order-lg-2">
+                        <h1 class="first-screen__title">
+                            Save your data storage here.
+                        </h1>
+                        <p class="first-screen__text">
+                            Data Warehouse is a data storage area that has been
+                            tested for security, so you can store your data here
+                            safely but not be afraid of being stolen by others.
+                        </p>
+                        <a class="button button-accent first-screen__button" href="#">Learn more</a>
+                    </div>
+                    <div class="col-2 order-lg-1">
+                        <img class="first-screen__img" src="./src/img/first-screen-img.svg" alt="data storage vector">
+                    </div>
+                </section>
+                <section class="pink-screen row" id="help">
+                    <div class="col-2 pink-screen__img-box">
+                        <img class="pink-screen__img" src="./src/img/second-screen-img.svg" alt="data storage bank">
+                    </div>
+                    <div class="col-2">
+                        <h2 class="pink-screen__title">
+                            We are a high-level data storage bank</h2>
+                        <p>
+                            The place to store various data that you can access
+                            at any time through the internet  and where you can carry it. 
+                        </p>
+                        <p>
+                            This very flexible storage area has a high level of security.
+                            To enter into your own data you must enter the password that
+                            you created when you registered in this Data Warehouse. 
+                        </p>
+                    </div>
+                </section>
+                <section class="features" id="features">
+                    <h2 class="features__title">Features</h2>
+                    <p class="features__text">
+                        Some of the features and advantages that we provide for those of
+                         you who store data in this Data Warehouse.
+                    </p>
+                    <div class="row row-with-gap">
+                        <div class="col-2 features__item">
+                            <img class="features__img" src="./src/img/card-image-1.svg" alt="">
+                            <div class="features__box bg-light-blue">
+                                <div class="features__info">
+                                    <h3 class="features__subtitle">Search Data</h3>
+                                    <p class="features__box-text">
+                                        Don’t worry if your data is very large, the Data
```

Comment:
The character U+2019 "’" could be confused with the character U+0060 "`", which is more common in source code. [Adjust settings]